### ISSUE TITLE
Earraping with a morgue

### ISSUE THREAD

**Issue by PortalVector** [tgstation/tgstation#35808]
[Round ID]: 83832

[Reproduction]: Spamming enter/get out his corpse (in a morgue)

(sorry for bad english (i'm french) but here is a video : https://youtu.be/itF5aN_mf8A).

(Put like a timer or eject the corpse when spamming).

**Comment by Sunoplyrus** [comment]
feature

**Comment by Xhuis** [comment]
The translation of that comment seems to be "I didn't know how to make this unspammable so I passed it off as a feature"

**Comment by Buggy123** [comment]
>The translation of that comment seems to be "I didn't know how to make this unspammable so I passed it off as a feature"

Yes, I do. See #34676, where I have a check to prevent sound spam. This was intentional, its supposed to be spammable. If someone is spamming it then _clone them you dickhead_, or just open the tray and take them out.

**Comment by BeeSting12** [comment]
@Buggy123 There's valid reasons to leave someone in a morgue without cloning them.

**Comment by Ratvar** [comment]
no there ain't, go gib/cremate them if they are a hussle

**Comment by Buggy123** [comment]
>There ain't, go gib/cremate them if they are a hussle, more raisons to make burgers

**Comment by Xhuis** [comment]
Just because you refuse to see other people's points of view doesn't make them invalid. As someone who plays medical decently often, I've had many rounds where the belongings of bodies in the morgue have saved my life and dramatically altered the round before.

i.e. RD suicided, so I morgued him as medical doctor, and when the AI started going rogue, I used his ID to rescue and revive the two heads of staff it killed, forcing it to go loud early and allowing me to escape because I knew was was about to happen.
